docker-compose.yml

```diff
@@ -16,6 +16,10 @@ services:
       JAEGER_ENDPOINT: http://jaeger-collector:14268/api/traces
       SearchApi__WebHooks__0__Name: dynadapter
       SearchApi__WebHooks__0__Uri: http://dynadapter/PersonSearch/MatchFound
+      SearchApi__WebHooks__0__EventName: MatchFound
+      SearchApi__WebHooks__0__Name: dynadapter
```

Comment:
__0__ => __1__ this represents the index of the array

Reply:
oops.. ommission.. was still testing though